spyder/widgets/sourcecode/tests/test_autoindent.py

```diff
@@ -153,6 +153,14 @@ def test_first_line():
          "test_commented_brackets"),
         ("s = a[(a['a'] == l) & (a['a'] == 1)]['a']\n", "s = a[(a['a'] == l) & (a['a'] == 1)]['a']\n",
          "test_balanced_brackets"),
+        ("a = (a  #  some comment\n", "a = (a  #  some comment\n     ", "test_inline_comment"),
+        ("len(a) == 1\n", "len(a) == 1\n", "test_balanced_brackets_not_ending_in_bracket"),
+        ("x = f(\n", "x = f(\n      ", "test_short_open_bracket_not_hanging_indent"),
+
+        pytest.mark.xfail(
+            ("foo = 1  # Comment open parenthesis (\n",
+             "foo = 1  # Comment open parenthesis (\n",
+             "test_comment_with parenthesis")),
```

Comment:
Yes, although this case of use is very rare, I was thinking address it in the rewrite for 3.2